src/Hosting/Hosting/src/GenericHost/GenericWebHostBuilder.cs

```diff
@@ -234,6 +234,7 @@ public IWebHostBuilder UseStartup([DynamicallyAccessedMembers(StartupLinkerOptio
             return this;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern", Justification = "We need to call a generic method on IHostBuilder.")]
```

Comment:
The case where this may be unsafe is if `IHostBuilder.ConfigureContainer` ever puts a `DynamicallyAccessedMembers` attribute on the `TContainerBuilder` type. The linker won't be able to see that you are filling in the TContainerBuilder with `containerType`.

Looking at the implementation In Microsoft.Extensions.Hosting, I don't see HostBuilder needing this, so I think this is safe.